src/AzureAuth/CommandMain.cs

```diff
@@ -376,14 +379,20 @@ private IAuthFlow AuthFlow()
                 // TODO: Really we need to get rid of Resource
                 var scopes = this.Scopes ?? new string[] { $"{this.authSettings.Resource}/.default" };
 
+                string promptHint = PromptHintPrefix;
+                if (string.IsNullOrEmpty(promptHint))

```

Comment:
If we set `promptHint` to be `PromptHintPrefix`, which is *always* a `const string` we've set beforehand, isn't it impossible for
```csharp
if (string.IsNullOrEmpty(promptHint) { ... }
```
to ever evaluate to `false`?

If so, does this not lead to a prompt hint of `AzureAuth: AzureAuth` every time? 🤔

Reply:
Oh, sorry, I misread it. I think what I meant is this will never be `true`, so we'll never include the users's prompt because this block can't execute. I could also just be confused here though...

Reply:
Sorry. I made a mistake here. it should be 
```C#
                string promptHint = PromptHintPrefix;
                if (!string.IsNullOrEmpty(this.authSettings.PromptHint))
                {
                    promptHint = $"{PromptHintPrefix}: {this.authSettings.PromptHint}";
                }
```

Reply:
I tried to make my code clear before committing, but made a huge mistake. Sad.

Reply:
Nothing to be sorry about, it seems like an easy enough mistake to make and it looks like you've corrected it. 😄 

My follow-up question is, this seems like the sort of thing that might have been caught with a test. Is there a way for us to test this so we don't regress? If so, let's try to add that in as well before we merge. ✅

Reply:
I slightly changed the code and added two unit tests.😁